system-addon/content-src/components/TopSites/TopSites.jsx

```diff
@@ -52,7 +54,8 @@ class TopSite extends React.Component {
           onUpdate={val => this.setState({showContextMenu: val})}
           site={link}
           index={index}
-          source={TOP_SITES_SOURCE} />
+          source={TOP_SITES_SOURCE}
```

Comment:
I think we still need this because the source gets passed as part of the `UserEvent` dispatched in `onClick`